components/home/DashboardButton.tsx

```diff
@@ -41,7 +43,30 @@ export default function DashboardButton({
           )}
         </Pressable>
       </View>
-    </Link>
+      </Link>
+    ) : (
+      <View>
+        <View style={styles.disabledBtn}>
```

Comment:
Instead of re-writing the whole labeledButton styles in disabledBtn for the sake of making small changes, we can spread the labeledButton  style into an inline style object and then add our css rule there.
I also think the opacity should be lower to indicate a disabled button.
```suggestion
           <View style={{ ...styles.labeledButton, opacity: 0.5 }}>
```

Reply:
Okay i will update MR for requested changes